pmd-core/src/main/java/net/sourceforge/pmd/renderers/CodeClimateRenderer.java

```diff
@@ -145,27 +145,27 @@ private int getRemediationPoints() {
     }
 
     private <T> String getBody() {
-        String result = "## " + rule.getName() + "\\n\\n" + "Since: PMD " + rule.getSince() + "\\n\\n" + "Priority: "
-            + rule.getPriority() + "\\n\\n"
-            + "[Categories](https://github.com/codeclimate/platform/blob/master/spec/analyzers/SPEC.md#categories): "
-            + Arrays.toString(getCategories()).replaceAll("[\\[\\]]", "") + "\\n\\n"
-            + "[Remediation Points](https://github.com/codeclimate/platform/blob/master/spec/analyzers/SPEC.md#remediation-points): "
-            + getRemediationPoints() + "\\n\\n" + cleaned(rule.getDescription());
+        StringBuilder result = new StringBuilder("## " + rule.getName() + "\\n\\n" + "Since: PMD " + rule.getSince() + "\\n\\n" + "Priority: "
```

Comment:
This stringbuilder uses a concatenation as an argument

Reply:
@oowekyala my bad. re-done&rebased.